test/parallel/test-tls-env-extra-ca-no-crypto.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+// Flags: --expose-internals
+
+const common = require('../common');
+const fixtures = require('../common/fixtures');
+const assert = require('assert');
+const { fork } = require('child_process');
+
+// This test ensures that trying to load extra certs won't throw even
+// though there is no crypto support.
```

Comment:
Maybe clarify that this test is written for `./configure --without-ssl` builds. (That's right, isn't it?